INSTALL.md

```diff
@@ -27,8 +27,8 @@ For more details, please refer configuration section from [README.md](README.md)
  ``` 
 ### RUN SDE Backend Locally
 #### Prerequisites
-- JDK18
-- Postgres 11.9.13
+- JDK17
+- Postgres 13.2.11
```

Comment:
Why do you specify Postgres v13 when in your helm chart you use chart version 11.x.x which potentially means Postgres v15?

Reply:
Sorry, I was wrong, Bitnami's Postgresql chart version 11.x.x means Postgresql v14. Although based on this [TRG](https://eclipse-tractusx.github.io/docs/release/trg-5/trg-5-07#aligning-dependency-versions) you should have already moved/upgraded to Postgres v15 which is chart version 12.x.x.

Reply:
We will be using Postgres v16. Can we use Postgresv13.X.X.

Reply:
Postgress upgraded to v15.